website/docs/docs/guides/migration-guide/upgrading-to-0-19-0.md

```diff
@@ -42,19 +43,21 @@ See [dbt#2961](https://github.com/fishtown-analytics/dbt/pull/2961) for full imp
 ## New and changed documentation
 
 ### Core
-- [dbt-artifacts](dbt-artifacts): The JSON artifacts produced by dbt—manifest, catalog, run results, and sources—are simpler to consume and more clearly documented.
-- [snapshots](snapshots) ([invalidate_hard_deletes](invalidate_hard_deletes)): If the config `invalidate_hard_deletes` is enabled, `dbt snapshot` will update records whose unique key no longer exist in the snapshot query. Should those uniquely identified records "revive," `dbt snapshot` will re-add them.
+- [dbt Artifacts](dbt-artifacts): The JSON artifacts produced by dbt—manifest, catalog, run results, and sources—are simpler to consume and more clearly documented.
+- [dbt Classes](dbt-classes#result-objects), [on-run-end Context](on-run-end-context#results): The `Result` object has a new schema,
+- [Statement blocks](statement-blocks): The `call statement` result `status` string is now a structured object named `response`.
+- [Snapshots](snapshots#invalidate_hard_deletes): If the config `invalidate_hard_deletes` is enabled, `dbt snapshot` will update records whose unique key no longer exist in the snapshot query. Should those uniquely identified records "revive," `dbt snapshot` will re-add them.
 - [YAML selectors](yaml-selectors) support a `description` property and record their expanded dictionary representations in the manifest.
-- [modules](modules): The regex python module, `re`, is available in dbt's Jinja context.
+- [Modules](modules): The regex python module, `re`, is available in dbt's Jinja context.
 
 #### State
 - [Understanding state](understanding-state): New docs outlining the conceptual background of state-informed runs, as well as the [known caveats](state-comparison-caveats) for state comparison. In v0.19.0, dbt is a little bit smarter at identifying `state:modified` "false positives" that previously resulted from env-based configurations in `dbt_project`.
 - [Defer](defer) has changed: Instead of deferring all unselected node references, dbt now defers an unselected node reference _if and only if_ it does not exist in the current environment. Tests can defer their upstream references as well. This better supports the "Slim CI" use case by addressing the current environment's resources across `seed`, `run`, and `test` steps.
-- [rpc](rpc): Added `state` and `defer` as arguments to RPC methods for which it is supported on the CLI.
+- [RPC](rpc): Added `state` and `defer` as arguments to RPC methods for which it is supported on the CLI.
 
 ### BigQuery
-- [bigquery-profile](bigquery-profile): dbt can connect via OAuth tokens (one-time or refresh), and it can use the default project when connecting via `gcloud` oauth.
+- [Bigquery profile](bigquery-profile): dbt can connect via OAuth tokens (one-time or refresh), and it can use the default project when connecting via `gcloud` oauth.
```

Comment:
```suggestion
- [BigQuery profile](bigquery-profile): dbt can connect via OAuth tokens (one-time or refresh), and it can use the default project when connecting via `gcloud` oauth.
```